### ISSUE TITLE
[WASM] Listview header and footer are not implemented

### ISSUE THREAD

**Issue by soriyathach** [unoplatform/uno#1979]
## Current behavior

The listview is not displayed at all if a header or footer is specified on a listview.

## Expected behavior

The header, footer and listview content should be displayed.

## How to reproduce it (as minimally and precisely as possible)

Seems like nothing is implemented on ListViewBase for wasm:
https://github.com/unoplatform/uno/blob/8dcc6f2d3aa5d57fd3ee96efe5fe73855a45a0da/src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.netstd.cs

## Environment

Nuget Package:

Package Version(s):
UNO.UI 1.46.196-dev.2440

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [x] 2017 (version: )
- [x] 2019 (version: 16.2.0)
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by michael-hawker** [comment]
Assume this is also for tracking it on the base of `ItemsPresenter`?

**Comment by jeromelaban** [comment]
> Assume this is also for tracking it on the base of `ItemsPresenter`?

@michael-hawker Not necessarily, the implementations of both are different (even in WinUI). Do you need it on either or both?

**Comment by michael-hawker** [comment]
@jeromelaban thanks for getting back to me, I just linked to our PR where we're adding new behavior to the SettingsExpander for this with `ItemsPresenter` as part of it being an `ItemsControl`.

We could do a bunch of extra work with a `StackPanel` and `ContentPresenter`s, but it's super simple to hook-in to the existing properties on the `ItemsPresenter` for this scenario.

Want me to spin up a separate issue for tracking `ItemsPresenter` specifically?

**Comment by jeromelaban** [comment]
Thanks for the update @michael-hawker, I'll create the issue for ItemsPresenter specifically.

**Comment by hawkerm** [comment]
Ah, forgot about this. Just tried adding a Header for some content and it's not working on WASM (though I do see the list view items still, unlike the original post). Just the header is missing.

**Comment by michael-hawker** [comment]
FYI, we have a new approach we're investigating in Windows Community Toolkit Labs for a future/modern `DataGrid`-light replacement: https://github.com/CommunityToolkit/Labs-Windows/discussions/415

This is relying on the use of the property `Header` on `ItemsControl`/`ListView` for the inner `ItemsPresenter` (see #10745 as well), so this prevents it from working with Uno/WASM.

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** `ListView` header and footer support has been implemented since this issue was filed — runtime tests covering header/footer behavior exist in `src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs` (e.g., `When_Header_DataContext`, `When_Footer_DataContext`). If specific header/footer scenarios are still broken, please file a new targeted issue.
